Spikes/movement/Character.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#ifndef CHARACTER_H
+#define CHARACTER_H
+
+#include "SFML/Graphics.hpp" 
+#include "SFML/OpenGL.hpp" 
+
+using namespace sf;
+
+
+
+class Character
+{
+private:
+	//Character
+	bool m_move;
+	Vector2f m_centre;
+	Vector2f m_pos;
+	float m_radius;
+	float m_speed;
+	float m_angle;
+	CircleShape m_shape;
+
+	//Target
+	Vector2f m_targetPos; 
+	float m_targetRadius;
+	float m_targetAngle;
```

Comment:
What is target radius? Make it apparent, also I would think that the target shape you are creating/drawing could be actually implemented elsewhere as opposed to the player/character.cpp. Instead character should maybe have a target pos instead.